docs/styleguide.md

```diff
@@ -0,0 +1,78 @@
+# Electron Documentation Styleguide
+
+Find the appropriate section for your task: [reading Electron documentation](#)
+or [writing Electron documentation](#).
+
+## Writing Electron Documentation
+
+These are the ways that we construct the Electron documentation.
+
+- Maximum one `h1` title per page.
+- Use `bash` instead of `cmd` in code blocks (because of syntax highlighter).
+- Doc `h1` titles should match object name (i.e. `browser-window` →
+  `BrowserWindow`).
+  - Hyphen separated filenames, however, are fine.
+- No headers following headers, add at least a one-sentence description.
+- Methods headers are wrapped in `code` ticks.
+- Event headers are wrapped in single 'quotation' marks.
+- No nesting lists more than 2 levels (unfortunately because of markdown
+  renderer).
+- Add section titles: Events, Class Methods and Instance Methods.
+- Use 'will' over 'would' when describing outcomes.
+- Events and methods are `h3` headers.
+- Optional arguments written as `function (required[, optional])`.
+- Optional arguments are denoted when called out in list.
+- Line length is 80-column wrapped.
+- Platform specific methods are noted in italics following method header.
+ - ```### `method(foo, bar)` _Mac_```
```

Comment:
We should use "OS X" instead of "Mac" here, there are probably other places need to be changed too.